### PR TITLE
Homework #2

### PR DESCRIPTION
**Pull request by Mad-Ninja**
@devbarden 
Added: user login, protect router with Passport JWT Strategy. In the process of writing controllers and interacting with the database.

### CODE REVIEW

File: controllers/accounts.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const {users} = require ('../database/users.js');
+
+
```
Comment (devbarden) [inline controllers/accounts.js:3]: extra empty line

File: controllers/login.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const bcrypt = require('bcrypt');
+const {getUserByEmail , users} = require('../database/users');
+
+const loginUser = (email,password) => {
+const user = getUserByEmail(email);
```
Comment (devbarden) [inline controllers/login.js:5]: very bad tabs in this function. please fix it because it hard to read

**Review by devbarden — COMMENTED** [review]
not bad but a lot of minor issues with code style. so please add eslint to your project to avoid this issues in future

File: controllers/accounts.js (hunk 1 above)
Comment (Mad-Ninja) [reply controllers/accounts.js]: fixed it

File: controllers/login.js (hunk 2 above)
Comment (Mad-Ninja) [reply controllers/login.js]: fixed it

**Comment by devbarden** [comment]
if you want you can merge it)

**Pull request merged**